### PR TITLE
adds fluentbit message

### PR DESCRIPTION
**Pull request by theoskolnik**
Includes a message around Fluent Bit upgrade given [customer feedback](https://hashicorp.slack.com/archives/CKE00RWUT/p1657825851969039).

### CODE REVIEW

File: v202207-1.md [hunk 1]
```diff
@@ -39,8 +39,8 @@
 ### Security
 
 1. The [External Vault](https://www.terraform.io/enterprise/requirements/data-storage/vault) policy has been updated to use specific API paths instead of wildcard matching.
-1. The version of the internally-managed Nomad server has been updated to to 1.3.1.
-1. Container updates have been adopted, addressing reported vulnerabilities (CVEs) in underlying packages / dependencies.
+2. The version of the internally-managed Nomad server has been updated to to 1.3.1.
+3. Container updates have been adopted, addressing reported vulnerabilities (CVEs) in underlying packages / dependencies. This change bumps the version of Fluent Bit in `tfe-fluent-bit` to use 1.9.5.
```
Comment (sudomateo) [inline v202207-1.md]: It's okay to leave the items as `1.`. The markdown rendering will give it real numbers.

```suggestion
1. The version of the internally-managed Nomad server has been updated to 1.3.1.
1. Container updates have been adopted, addressing reported vulnerabilities (CVEs) in underlying packages / dependencies. This change bumps the version of Fluent Bit in `tfe-fluent-bit` to 1.9.5.
```

**Review by sudomateo — APPROVED** [review]
Minor formatting changes. Approving to not block.

**Review by nathan-lacey — COMMENTED** [review]
You have "to to" stated twice on line 42. Otherwise, looks good.

**Review by nathan-lacey — APPROVED** [review]
👍

**Comment by theoskolnik** [comment]
Nathan caught some more typos pre- my change, I'll update those as well!

**Pull request merged**